pcsx2/System/SysThreadBase.cpp

```diff
@@ -199,48 +209,55 @@ void SysThreadBase::PauseSelfDebug()
 //
 void SysThreadBase::Resume()
 {
-	if( IsSelf() ) return;
-	if( m_ExecMode == ExecMode_Opened ) return;
+	if (IsSelf())
+		return;
+	if (m_ExecMode == ExecMode_Opened)
+		return;
 
-	ScopedLock locker( m_ExecModeMutex );
+	ScopedLock locker(m_ExecModeMutex);
 
 	// Implementation Note:
 	// The entire state coming out of a Wait is indeterminate because of user input
 	// and pending messages being handled.  So after each call we do some seemingly redundant
 	// sanity checks against m_ExecMode/m_Running status, and if something doesn't feel
 	// right, we should abort; the user may have canceled the action before it even finished.
 
-	switch( m_ExecMode.load() )
+	switch (m_ExecMode.load())
 	{
-		case ExecMode_Opened: return;
+		case ExecMode_Opened:
+			return;
 
 		case ExecMode_NoThreadYet:
 		{
 			Start();
-			if( !m_running || (m_ExecMode == ExecMode_NoThreadYet) )
+			if (!m_running || (m_ExecMode == ExecMode_NoThreadYet))
 				throw Exception::ThreadCreationError(this);
-			if( m_ExecMode == ExecMode_Opened ) return;
+			if (m_ExecMode == ExecMode_Opened)
+				return;
 		}
-		// fall through...
+			// fall through...
```

Comment:
Can you get rid of the space, dunno why Clang did that. maybe better still replace it with [[fallthrough]];

Reply:
Done